sema/commons/glob/globbery.py

```
@@ -0,0 +1,88 @@
+from abc import ABC, abstractmethod
+from pathlib import Path
+from typing import Any, Dict, List
+
+
+def getMatchingGlobPaths(
+    root: Path,
+    includes: List[str] = ["**/*"],
```

Ruff (0.8.2)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)